airflow/jobs/local_task_job.py

```diff
@@ -161,6 +191,10 @@ def handle_task_exit(self, return_code: int) -> None:
         """
         # Without setting this, heartbeat may get us
         self.terminating = True
+
+        if abs(return_code) == signal.SIGSEGV:  # Handle signal.SIGSEGV and psutil._psposix.Negsignal.SIGSEGV
```

Comment:
Why the `abs`? Positive and negative return codes are different values. If you want to remove trash bits over 256 you need a bit mask instead.

Reply:
Oh... I'm understand that I mix up kill signals (SIGSEGV = 11) and return_code which, I thought, depends on `task_runner` and for default `StandardTaskRunner` return code when task failed with Segmentation Fault in this case -11 and might be different for CgroupTaskRunner (I assume that 139 but need to check)

So might be better move this log message from `airflow/jobs/local_task_job.pyy` to a `airflow/task/task_runner/standard_task_runner.py` and `airflow/task/task_runner/cgroup_task_runner.py`? WDYT?

Reply:
Exit codes except 0 and 1 are implementation-defined, and I’m not sure what `cgexec` returns for errors, so some trail-and-errors would probably be needed. If they do return different values for this situation, it’d make sense to put the log in runners.